wrangler-core/src/test/java/io/cdap/directives/column/SetTypeTest.java

```diff
@@ -154,9 +155,9 @@ public void testToDecimal() throws Exception {
         .add("bytes_col", new byte[] {0, 0, 0, 0, 39, 16}).add("decimal_col", new BigDecimal("10000"))
     );
     String[] directives = new String[] {
-      "set-type str_col decimal", "set-type int_col Decimal", "set-type double_col DECIMAL",
-      "set-type short_col DECIMAL", "set-type long_col Decimal", "set-type float_col DECIMAL",
-      "set-type bytes_col decimal", "set-type decimal_col decimal"
+      "set-type str_col decimal 0", "set-type int_col Decimal 0", "set-type double_col DECIMAL 1",
```

Comment:
We should also be supporting just, 'set-type str_col decimal' without any scale.

Reply:
yes, that is supported